test/conformance/source/environment.cpp

```diff
@@ -42,15 +67,6 @@ std::ostream &operator<<(std::ostream &out,
     return out;
 }
 
-std::ostream &operator<<(std::ostream &out, const ur_device_handle_t &device) {
```

Comment:
This has been removed since we are importing a few more headers which define this elsewhere.